fleetspeak/src/server/comms.go

```diff
@@ -45,6 +46,16 @@ type commsContext struct {
 	s *Server
 }
 
+func randUint64() uint64 {
+	var b [8]byte
+	_, err := rand.Read(b[:])
+	if err != nil {
```

Comment:
You can use the usual error-handling pattern here, no (place this in the previous line)?

Reply:
Done.